src/components/SliderWrapper/index.js

```diff
@@ -8,93 +8,69 @@ import ArrowRight from "../../icons/ArrowRight";
-          <div
-            onClick={() =>
-              position !== 0 && onClick(position + WIDTH_IMG_SLIDER)
-            }
-            className={classNames(
-              { "arrow-unactive": position === 0 },
-              { arrow: true }
-            )}
-          >
-            <ArrowLeft />
-          </div>
-        )}
-        <div className="client__list-wrapper">
-          <div style={{ marginLeft: position }} className="client__list-slider">
-            <ClientList data={filterData} startPath={startPath} />
-          </div>
+  const dataLength = filterData.length;
+  return (
+    <div className="customer-groups__slider">
+      {dataLength > maxItemLength && (
+        <div
+          onClick={() => position !== 0 && onClick(position + WIDTH_IMG_SLIDER)}
+          className={classNames("arrow", { "arrow-unactive": position === 0 })}
+        >
+          <ArrowLeft />
+        </div>
+      )}
+      <div className="client__list-wrapper">
+        <div style={{ marginLeft: position }} className="client__list-slider">
+          <ClientList data={filterData} startPath={startPath} />
         </div>
-        {dataLength > maxItemLength && (
-          <div
-            className={classNames(
-              {
-                "arrow-unactive":
-                  position ===
-                  maxItemLength * WIDTH_IMG_SLIDER -
-                    dataLength * WIDTH_IMG_SLIDER
-              },
-              { arrow: true }
-            )}
-            onClick={() =>
-              position !==
-                maxItemLength * WIDTH_IMG_SLIDER -
-                  dataLength * WIDTH_IMG_SLIDER &&
-              onClick(position - WIDTH_IMG_SLIDER)
-            }
-          >
-            <ArrowRight />
-          </div>
-        )}
       </div>
-    );
-  }
+      {dataLength > maxItemLength && (
+        <div
+          className={classNames("arrow", {
+            "arrow-unactive":
+              position ===
+              maxItemLength * WIDTH_IMG_SLIDER - dataLength * WIDTH_IMG_SLIDER
+          })}
+          onClick={() =>
+            position !==
+              maxItemLength * WIDTH_IMG_SLIDER -
+                dataLength * WIDTH_IMG_SLIDER &&
+            onClick(position - WIDTH_IMG_SLIDER)
+          }
+        >
+          <ArrowRight />
+        </div>
+      )}
+    </div>
+  );
+};
 
-  changeState = () => {
-    if (this.props.windowWidth > 1111) {
-      return this.setState({
-        maxItemLength: 4
-      });
-    }
-    if (869 < this.props.windowWidth && this.props.windowWidth <= 1111) {
-      return this.setState({
-        maxItemLength: 3
-      });
-    }
-    if (619 < this.props.windowWidth && this.props.windowWidth <= 869) {
-      return this.setState({
-        maxItemLength: 2
-      });
-    }
-    if (360 < this.props.windowWidth && this.props.windowWidth <= 619) {
-      return this.setState({
-        maxItemLength: 1
-      });
-    }
-  };
-}
+const changeState = (width, func) => {
```

Comment:
function which set in props what will be called in different case by default name callback or cb not `func`
just rename `func` -> `cb`